airbyte-integrations/connectors/source-public-apis/source_public_apis/spec.yaml

```diff
@@ -0,0 +1,7 @@
+documentationUrl: https://api.publicapis.org
```

Comment:
This is not correct.

Reply:
changed it to `https://docs.airbyte.com/integrations/sources/public-apis`